fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -338,7 +345,14 @@ class PythonUtils {
         
     def readAttributeMethodHL(PogoDeviceClass cls, Attribute attribute) '''
         def read_«attribute.name»(self):
-                «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»«protectedAreaHL(cls, attribute.name + "_read", "return " + attribute.defaultValueHL, false)»«ELSE»return «attribute.defaultValueDim»«ENDIF»
+                «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»
+                «openProtectedAreaHL(cls, attribute.name + "_read")»
+                """Return the «attribute.name» attribute."""
+                return «attribute.defaultValueHL»
+                «closeProtectedAreaHL(cls, attribute.name + "_read")»
+                «ELSE»
+                return «attribute.defaultValueDim»
+                «ENDIF»
```

Comment:
I notice that the indentation of the write and read methods differs slightly in the Github view.  Do the resultant files have the correct indentation?

Maybe some lines have tab characters and others have spaces?  Would be good make all the formatting the same (my preference is to use spaces only. 4 spaces per TAB, as per PEP8).

Reply:
Yes the resultant files have the same indentation. 
I have aligned both the methods with same indentation in formatting as well. I will add that change as part of next pull request.